### PR TITLE
old project review

### PR DESCRIPTION
**Pull request by bhushan354**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,8 +11,8 @@ <h1 id="title">Dr. Norman Borlaug</h1>
...
         <p id="img-caption">Dr. Norman Borlaug, father of the Green Revolution</p>
       </div>
-      <p id="tribute-info">Dr. Norman Borlaug was an American agronomist and humanitarian who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution. Borlaug was awarded multiple honors for his contributions to agriculture, including the Nobel Peace Prize, the Presidential Medal of Freedom, and the Congressional Gold Medal.</p>
-      <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Learn more about Dr. Norman Borlaug</a>
+      <p id="tridbute-info">Dr. Norman Borlaug was an American agronomist and humanitarian who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution. Borlaug was awarded multiple honors for his contributions to agriculture, including the Nobel Peace Prize, the Presidential Medal of Freedom, and the Congressional Gold Medal.</p>
+      <a id="tridbute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Learn more about Dr. Norman Borlaug</a>
```
Comment (bhushan354) [inline index.html]: - Though everything is good in your code but there is small typo . i suggest you to change 'tridbute' to 'tribute'.

File: style.css [hunk 2]
```diff
@@ -10,6 +10,8 @@
   }
   
   #image {
+    /* display: flex;
+    margin-top: 0cap; */
```
Comment (bhushan354) [inline style.css]: - Everything is perfect in your css code , good job!!. but i would suggest that you should not keep unused code or comments in your codebase . in order to make your code clean try deleting unused code

**Review by bhushan354 — COMMENTED** [review]
# Changes required ♻️ 
- make sure you use proper linters .

-Good job in writing code however you need few more changes before moving forward make sure you check my comments below and make all required changes.

-submit for re-review when you're done
thanks 😃

**Review by skyv26 — APPROVED** [review]
## Hi @bhushan354, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 


## Highlights
- Good **PR** title ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Polite and clear comments ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    margin: auto;
+    text-align: center;
+  }
+  
+  #img-div {
+    max-width: 600px;
+    margin: auto;
+  }
+  
+  #image {
```
Comment (Mike47ip) [inline style.css]: Great work!

File: style.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    /* display: flex;
+    margin-top: 0cap; */
+    display: block;
+    margin: auto;
+    max-width: 100%;
+    height: auto;
+  }
+  
+  #img-caption {
```
Comment (Mike47ip) [inline style.css]: Clean code

**Review by Mike47ip — COMMENTED** [review]
Nothing to change here. Keep up the good work.

**Pull request closed without merging**